CMakeLists.txt

```diff
@@ -155,9 +154,7 @@ target_link_libraries(fv3atm PUBLIC fv3dycore
                                     stochastic_physics
                                     fms)
 
-target_link_libraries(fv3atm PUBLIC nemsio::nemsio
-                                    w3emc::w3emc_d
-                                    w3nco::w3nco_d
+target_link_libraries(fv3atm PUBLIC w3nco::w3nco_d
                                     sp::sp_d
```

Comment:
Can you try if you can remove sp lib? I realized that we have splat subroutine in module_wrt_grid_comp.F90, so maybe we do not need the sp lib in fv3atm and ufs.

Reply:
Physics needs sp library:
```
FV3/ccpp/physics/libccppphys.a(sfcsub.F.o): In function `setrmsk_':
/work/noaa/fv3-cam/djovic/ufs/remove_nemsio/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:6103: undefined reference to `splat_'
FV3/ccpp/physics/libccppphys.a(sfcsub.F.o): In function `ga2la_':
/work/noaa/fv3-cam/djovic/ufs/remove_nemsio/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:6614: undefined reference to `splat_'
FV3/ccpp/physics/libccppphys.a(sfcsub.F.o): In function `gaulat_':
/work/noaa/fv3-cam/djovic/ufs/remove_nemsio/ufs-weather-model/FV3/ccpp/physics/physics/sfcsub.F:6842: undefined reference to `splat_'
make[2]: *** [ufs_model] Error 1
```
Maybe ccpp-physics should depend on sp library instead of fv3atm.

Reply:
I agree, it's good to specify ccpp physics dependency on sp library. maybe when all the fixed files move to native grid we can then get rid of the sp dependency.